src/main/java/uk/gov/digital/ho/hocs/casework/api/CaseDocumentService.java

```diff
@@ -83,41 +100,12 @@ public S3Document getDocumentPdf(UUID documentUUID) {
         return document;
     }
 
-    private void enrichDocumentsResponse(GetDocumentsResponse getDocumentsResponse, CaseData caseData) {
-        String caseType = caseData.getType();
-        Map<String, String> data = caseData.getDataMap(objectMapper);
-
+    private void enrichDocumentsResponse(GetDocumentsResponse getDocumentsResponse, String fieldName, String caseType) {
```

Comment:
I have started to remove CaseData and StageWithCaseData objects from these methods, as we StageWithCaseData shouldn't exist, these methods should be agnostic.